lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2230,6 +2258,47 @@ public function get_styles_for_block( $block_metadata ) {
 			}
 		}
 
+		// If there are style variations, generate the selectors for them, including any feature selectors.
```

Comment:
It looks like this code block generates the whole CSS declarations, and not just the selectors. Should we update the comment to reflect that?